packages/client/tests/e2e/_utils/standard.cmd.sh

```diff
@@ -1,19 +1,35 @@
-#!/bin/sh
+#!/bin/bash
```

Comment:
```suggestion
#!/bin/bash
set -eio pipefail
```

I prefer to have this in most of bash scripts for safety & less painful debugging. [Explanation](https://vaneyckt.io/posts/safer_bash_scripts_with_set_euxo_pipefail/).

Would also allow to skip explicit `exit` at the end of this script, as pipe and individual command errors would be autoamtically proopogated.